packages/transit-vehicle-overlay/README.md

```diff
@@ -1,23 +1,44 @@
-            after a true will result in an empty popup artifact (you can see this on Storybook) 
-   hasTooltip=optional boolean (default true) ... show tooltip on desktop (leaflet's idea of what is desktop vs mobile)
-   visible
-/>
+<Vehicles name="optional string: This name will appear in the leaflet layer
+switcher. No string value for name will not list the vehicles in the layer
+switcher. The component can also be controlled by the visbile property, where
+layer display and interval fetches are controlled programmatically (see
+storybook for examples)" visible="true or false: programmatic prop control over
+the display and fetch of vehicle data" geometryUrl="base URL to GeoJSON transit
+pattern data -- the url is filled out by tracking vehicle records' pattern ids,
+etc" vehicleUrl="base URL to vehicle data service" vehicleQuery="optional
+string: (default 'routes/all'): eg: 'routes/90,100,190,200,290' will show just
+MAX route vehicles " refreshDelay="how often are new vehicles fetched from the
+service (default 5000 ms)." tracked="optional string: either a blockId (better)
+or tripId used to follow a given vehicle's progress" note: the tracked vehicle
+must be in the list of queried vehicles, and running the specified trip ... for
+example, a vehicle that's on a layover might not yet be running with the tripId
+you'd want to track using blockId is better than trip, since a blockId will
+track a vehicle over multiple trips recenterMap="optional boolean (default true)
+pans the map to the selected vehicle as it move along its trip"
+panOffsetX="shift map center point left or right X pixels of the tracked vehicle
+(see recenterMap == true)" panOffsetY="shift map center point up or down Y
+pixels of the tracked vehicle (see recenterMap == true)" hasPopup="optional
+boolean (default true) ... show vehicle marker popups" note: advice is to set
+once prior to the component being instantiated, since resetting this to false
+after a true will result in an empty popup artifact (you can see this on
+Storybook) hasTooltip=optional boolean (default true) ... show tooltip on
+desktop (leaflet's idea of what is desktop vs mobile) color="optional string
+(defaults to black): changes the color of the vehicle markers"
+highlightColor="optional string (defaults to cyan/blue): for tracked vehicle
+marker and the route geom the vehicle is proceeding along"
+lowlightColor="optional string (defaults to grey): for showing the partial route
+geometry previously travelled by the tracked vehicle" highlight="line geom css
+for tracked route" lowlight="line geom css for tracked route previously
+travelled" onTrackedVehicleUpdate="function callback to share the tracked
+vehicle record with the parent ... a new record is sent on each fetch, thus
+sharing position and other changes" onVehicleListUpdate="function callback to
+share the list of vehicles fetched from the service every N seconds (see
+refreshDelay)" />
```

Comment:
This formatting is kind of hard to read. Can it be indented and have nicer line breaks?

Reply:
Grr.  That formatting wasn't what I was going after ... fix (hopefully) on the way.
BTW, thanks for the review, Evan.

Reply:
Hopefully we don't lose the formatting going forward (it's not great, but not totally unreadable): https://github.com/OpenTransitTools/otp-ui/blob/vehicles_dev/packages/transit-vehicle-overlay/README.md